src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -53,6 +61,11 @@ static bool SendQueryInSingleRowMode(PGconn *connection, char *query);
 static bool StoreQueryResult(MaterialState *routerState, PGconn *connection,
 							 TupleDesc tupleDescriptor, int64 *rows);
 static bool ConsumeQueryResult(PGconn *connection, int64 *rows);
+static void RegisterRouterExecutorXactCallbacks();
```

Comment:
I don't know how this got back in. A compiler complained at me about it at one point and I fixed it. Must have been during rebase; sorry.